doc/pcieinfo_design.md

```diff
@@ -0,0 +1,81 @@
+## Motivation
+Add a PCIe Diag tool for SONiC. This tool including three commands    
+### Commands
+
+    show platform pcieinfo     -----> Show current device PCIe info
+    show platform pcieinfo -c  -----> Check whether the PCIe info is correct   
+    pcieutil pcie_generate     -----> Generate an PCIe info congfig file
+
+## Implementation 
+### show utility update
+New item under menu `platform` in `show/main.py`  
+It will execute `pcieutil [options]`command
+
+**Syntax**
+
+     show  platform pcieinfo         ---->  pcieutil pcie_show
+     show  platform pcieinfo --check ---->  pcieutil pcie_check
```

Comment:
are '-c' and '--check' both working as valid parameter? If so, please list both here and above.